evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -13,8 +13,18 @@ <h4 class="{% if not forloop.first %} mt-3 {% elif contributor %} me-auto {% end
                 >
                     {% blocktranslate %}I can't give feedback about this contributor{% endblocktranslate %}
                 </button>
+                <confirmation-modal class="mark-no-answer-modal" confirm-button-class="btn-danger" id="mark-no-answer-modal">
```

Comment:
The id is not used - and wrong, because it is duplicated when there is more than one contributor